mm2src/rpc_task/src/manager.rs

```diff
@@ -72,6 +72,7 @@ impl<Task: RpcTask> RpcTaskManager<Task> {
             // `task.run(&task_handle)`.
             match task_result {
                 Some(task_result) => {
+                    // FIXME: If the result is an error, shouldn't we cancel the task at this point? (check the e.g. in the same commit).
```

Comment:
I think so, should be an opened issue and done in a separate PR though to examine all the potential issues with this change.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/issues/2503